scripts/cleanup.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+import argparse
+import glob
+import gzip
+import os
+from datetime import datetime, timedelta
+from typing import Dict, Iterable, List
+
+# Cleanup for rageshake server output files
+#
+# Example usage:
+#
+# ./cleanup.py --dry-run --path /home/rageshakes/store --max-days 100 element-auto-uisi:90
+#
+# No dependencies required beyond a modern python3.
+
+
+class Cleanup(object):
+    def __init__(
+        self,
+        limits: Dict[str, int],
+        days_to_check: List[int],
```

Comment:
You're calling this with an `Iterable`, so this is a typing violation.

```suggestion
        days_to_check: Iterable[int],
```

Generally it might be a good idea to run the whole thing through `mypy`.

Reply:
I did run `mypy`, `black` and `isort` before i committed and there were no errors, i may have just driven mypy wrong. Will try to get it to error before i fix it.

Reply:
can definitely make mypy complain about other things in the file when I run it on commandline; however, pycharm complains correctly.. No idea; fixed anyway